test/ts/tsconfig.json

```diff
@@ -0,0 +1,22 @@
+{
+  "compilerOptions": {
+    "strict": true,
+    "noEmit": true,
+    "jsx": "react",
+    "jsxFactory": "h",
+    "baseUrl": "../..",
+    "typeRoots": [
+      "node_modules/@types/node/index.d.ts"
```

Comment:
Is this also requires `@types/node` to be in deps? Tests are not passing...

Reply:
Let me to review the tests again...